app/assets/stylesheets/blacklight/_bookmark.scss

```diff
@@ -3,6 +3,10 @@ label.toggle-bookmark {
 	min-width: 8.5em;
 }
 
+.toggle-bookmark {
+  cursor: pointer;
```

Comment:
Can you move this up into `label.toggle-bookmark`?  Unfortunately, BL uses `.toggle-bookmark` class twice for different levels in the same hierarchy.

Reply:
I did that first. What I found is the checkbox looses the pointer if I do that.

Reply:
Or do you mean move it to be 

.toggle-bookmark {
cursor: pointer;
label {
}
}

Reply:
@dnoneill if not `label.toggle-bookmark` can you put it in `div.toggle-bookmark`?  This will help me understand where I goes when I rebase https://github.com/projectblacklight/blacklight/pull/3306

Reply:
Yup.